kotlinx-coroutines-core/build.gradle

```diff
@@ -110,12 +110,26 @@ compileKotlinMetadata {
 // :KLUDGE: Idea.active: This is needed to workaround resolve problems after importing this project to IDEA
 def configureNativeSourceSetPreset(name, preset) {
     def hostMainCompilation = project.kotlin.targetFromPreset(preset).compilations.main
+    // Kotlin 1.3.x: Look for platform libraries in "compile" configuration
     def compileConfiguration = configurations[hostMainCompilation.compileDependencyConfigurationName]
-    def hostNativePlatformLibs = files(provider {
-        compileConfiguration.findAll {
-            it.path.endsWith(".klib") || it.absolutePath.contains("klib${File.separator}platform")
+    // Kotlin 1.4.x: Look for platform libraries in "implementation" for default source set
+    def implementationConfiguration = configurations[hostMainCompilation.defaultSourceSet.implementationMetadataConfigurationName]
+    // Now find the libraries:
+    // Kotlin 1.3.x: Find all platform libs
+    // Kotlin 1.4.x: Finds platform libs & stdlib, but platform declarations are still not resolved due to IDE bugs
```

Comment:
Do we have the links to these bugs to allow future reviews to check if they are resolved? If so, it'd be nice to add extra comment lines for them IMO.

Reply:
They will be fixed before 1.4 release and affect only HMPP projects anyway.